db/scripts/V1__default.sql

```diff
@@ -1,5 +1,5 @@
 PRAGMA foreign_keys = OFF;
-BEGIN TRANSACTION;
+--BEGIN TRANSACTION;
```

Comment:
WHy do you remove transaction?

Reply:
Flyway runs each migration in a separate transaction